src/sonymn_int.cpp

```diff
@@ -806,6 +806,31 @@ namespace Exiv2 {
         return tagInfoFp_;
     }
 
+    //! Sony Tag 9403 SonyMisc1
+    constexpr TagInfo SonyMakerNote::tagInfoSonyMisc1_[] = {
+        {0x05,  "CameraTemperature", N_("Camera temperature"),
+        		N_("Internal camera temperature (in degrees Celsius)"),
+				sonyMisc1Id, makerTags, signedByte, -1, printTemperatureInDegC},
+        // End of list marker
+        {0xffff, "(UnknownSonyMisc1Tag)", "(UnknownSonyMisc1Tag)",
+        		 "(UnknownSonyMisc1Tag)",
+				 sonyMisc1Id, makerTags, unsignedByte, -1, printValue}
+    };
+
+    const TagInfo* SonyMakerNote::tagListSonyMisc1()
+    {
+        return tagInfoSonyMisc1_;
+    }
+
+    std::ostream& SonyMakerNote::printTemperatureInDegC(std::ostream& os, const Value& value, const ExifData*)
+    {
+        if (value.count() != 1)       os << "(" << value << ")";
+        else                          os << value << " °C";
```

Comment:
This indentation is a bit weird. Please could you make it consistent with the rest of the file?

Reply:
Thanks for looking at this. I have made the change and will use the usual format in future.